packages/webapp/src/__tests__/DepositOrWithdrawModal.test.ts

```
@@ -1,25 +1,35 @@
+import { render, fireEvent, screen, waitFor, cleanup } from '@testing-library/svelte';
 import DepositOrWithdrawModal from '$lib/components/DepositOrWithdrawModal.svelte';
 import { transactionStore } from '@rainlanguage/ui-components';
-import { signerAddress } from '$lib/stores/wagmi';
 import { readContract, switchChain } from '@wagmi/core';
 
 import type { ComponentProps } from 'svelte';
 import { getVaultApprovalCalldata } from '@rainlanguage/orderbook/js_api';
 import { getVaultDepositCalldata } from '@rainlanguage/orderbook/js_api';
+import { get } from 'svelte/store';
 
 export type ModalProps = ComponentProps<DepositOrWithdrawModal>;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)